common/SC_Filesystem_unix.cpp

```diff
@@ -32,6 +32,11 @@
 // system
 #    include <glob.h> // ::glob, glob_t
 
+// workaround for musl not yet supporting GLOB_TILDE
+#ifndef GLOB_TILDE
+#    define GLOB_TILDE 0
+#endif
```

Comment:
to satisfy our linter, these lines need to be indented:
```cpp
#    ifndef GLOB_TILDE
#        define GLOB_TILDE 0
#    endif
```

Reply:
Done.